.changeset/friendly-planets-care.md

```diff
@@ -0,0 +1,9 @@
+---
+"saleor-app-search": minor
+---
+
+Improved attributes mapping. Now Algolia will receive better products data:
+- Added `allVariants` field which is *an array of variant id strings*. It will only contain other variants, so if the array is empty, it means the variant is the only one. It can be quickly used to count alternative variants or to reference them by ID.
```

Comment:
If we filter out one variant, maybe `otherVariants` would be a better name? I would expect to have *all* variants in such a field.
Solutions:
- don't filter the variants list, or
- rename the field

Reply:
Good catch, I think I will rename to "other variants", it doesn't make sense to include own variant because you are already in its context

Reply:
Changed